spark/src/test/scala/org/apache/spark/sql/delta/managedcommit/ManagedCommitTestUtils.scala

```diff
@@ -72,6 +72,26 @@ trait ManagedCommitTestUtils
     }
   }
 
+  /**
+   * Run the test against a [[TrackingCommitOwnerClient]] with backfill batch size =
+   * `batchBackfillSize`
+   */
+  def testWithTrackingInMemoryCommitOwner(
```

Comment:
The other users of this utils class won't fully understand different commit owner implementations etc. Should we call it `def testWithManagedCommits(` instead?

Reply:
Sounds good, updated.